internal/controller/velero_test.go

```
@@ -3404,3 +3404,268 @@ func TestDPAReconciler_buildVeleroDeploymentWithAzureWorkloadIdentity(t *testing
+				"app.kubernetes.io/name": "velero",
+				"custom-label":           "custom-value",
+				"team":                   "backup",
+			},
+		},
+		{
+			name: "protected labels are filtered from user input",
+			dpa: &oadpv1alpha1.DataProtectionApplication{
+				Spec: oadpv1alpha1.DataProtectionApplicationSpec{
+					ResourceLabels: map[string]string{
+						"app.kubernetes.io/name":       "user-override", // should be ignored
+						"app.kubernetes.io/managed-by": "user-manager", // should be ignored
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gci)